docs/api/ref/Channel.md

```diff
@@ -431,6 +433,99 @@ catch (Exception ex)
 - [`broadcast`](#broadcast)
 - [`addContextListener`](#addcontextlistener)
 
+### `clearContext`
+
+<Tabs groupId="lang">
+<TabItem value="ts" label="TypeScript/JavaScript">
+
+```ts
+public clearContext(contextType?: string): Promise<void>;
+```
+
+</TabItem>
+<TabItem value="dotnet" label=".NET">
+
+```csharp
+Task ClearContext(string? contextType);
+```
+
+</TabItem>
+</Tabs>
+
+Used to clear the specified context type if provided, otherwise, clear all contexts. Desktop Agent MUST update internal context. Any future calls to [`getCurrentContext`](#getcurrentcontext) and any new joiners on that channel (through [`joinUserChannel`](DesktopAgent#joinUserChannel) or [`addContextListener`](DesktopAgent#addContextListener)) will not receive anything for specified context type or all contexts accordingly. 
+Desktop Agent MUST update all channel listeners subscribed to the  [`fdc3.nothing`](../../context/ref/Nothing.md/#nothing) type. If `contextType` is provided, then `subType` will be specified, otherwise, it is ommitted. 
```

Comment:
```suggestion
Desktop Agents MUST also immediately broadcast the  [`fdc3.nothing`](../../context/ref/Nothing.md/#nothing) type, which applications may listen to to be notified of the cleared context. If a `contextType` was provided, then the `subType` field will be be set to that type, otherwise, it is omitted. 
```

Reply:
Question, Kris, should I change "Desktop Agent" to "Desktop Agents" everywhere in my descriptions?